harness/determined/deploy/gcp/gcp.py

```diff
@@ -277,7 +277,10 @@ def delete(configs: Dict, env: Dict) -> None:
     command = ["terraform", "destroy"]
 
     command += ["-input=false"]
-    command += ["-auto-approve"]
+
+    if auto_approve:
+        command += ["-auto-approve"]
```

Comment:
maybe we should also respect this setting in `def terraform_apply` line 124 in this file?

Reply:
We don't really need an auto-approve flag for this because the use case is that we don't want things to be accidentally deleted. But we don't really need manual approval for requesting any resources.

Reply:
should we then call the user-facing flag a more generic `-y` / `--yes`, so it doesn't look like it's literally used for terraform?

Reply:
Will use `no-prompt`